### DISCUSSION TITLE
UTF-8 characters missing in browser

### DISCUSSION THREAD

**Discussion by xrth87** [AvaloniaUI/Avalonia#13487] [Q&A] [answered]
Hello, 
today I played a bit around with avalonia in the browser. It worked really well. 

But fonts seemed to be different and some special characters were not shown like arrows on buttons(→, ↧  and ←). How can I fix this?

![grafik](https://github.com/AvaloniaUI/Avalonia/assets/94718532/318ebc05-df51-44df-805f-d7294d8810b7)

In Code I just put the arrows in the content of the button.
`<Button Classes="image_button" Margin="5 5 5 0" Content="→"/>`

Thanks for any advise!

**Comment by maxkatz6** [comment, marked as answer]
It looks like you are using default monotype font that browser provides. You need to include any custom font or add `.UseInterFont()` to the Avalonia AppBuilder to use Inter font (shipped with our nuget packages).

**Reply by xrth87** [reply]
Thank you, the method seems to be`WithInterFont` now.
